packages/xo-web/src/common/xo/index.js

```diff
@@ -2017,8 +2017,15 @@ export const runBackupNgJob = ({ force, ...params }) => {
 
 export const listVmBackups = remotes => _call('backupNg.listVmBackups', { remotes: resolveIds(remotes) })
 
-export const restoreBackup = (backup, sr, startOnRestore) => {
+export const restoreBackup = (backup, sr, options) => {
```

Comment:
```js
export const restoreBackup = (backup, sr, { generateNewMacAddresses = false, startOnRestore = false } = {}) => {
```

Reply:
But if we do not send the options params eg :`return restoreBackup(backup,sr)`, it will throw an error `Cannot read property 'generateNewMacAddresses' of undefined`